new_version.sh

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python3
```

Comment:
More complete, latest script this is coming from:
https://github.com/ECP-WarpX/WarpX/blob/development/Tools/Release/newVersion.sh